src/Integrator/Flame.cpp

```diff
@@ -139,81 +141,81 @@ Flame::Parse(Flame& value, IO::ParmParse& pp)
+            pp_query_default("pressure.a1", value.pressure.arrhenius.a1, 0.0); // Surgate heat flux model paramater - AP
+            pp_query_default("pressure.a2", value.pressure.arrhenius.a2, 0.0); // Surgate heat flux model paramater - HTPB
+            pp_query_default("pressure.a3", value.pressure.arrhenius.a3, 0.0); // Surgate heat flux model paramater - Total
+            pp_query_default("pressure.b1", value.pressure.arrhenius.b1, 0.0); // Surgate heat flux model paramater - AP
+            pp_query_default("pressure.b2", value.pressure.arrhenius.b2, 0.0); // Surgate heat flux model paramater - HTPB
+            pp_query_default("pressure.b3", value.pressure.arrhenius.b3, 0.0); // Surgate heat flux model paramater - Total
+            pp_query_default("pressure.c1", value.pressure.arrhenius.c1, 0.0); // Surgate heat flux model paramater - Total
+            pp_query_default("pressure.mob_ap", value.pressure.arrhenius.mob_ap, 0); // Whether to include pressure to the arrhenius law
+            pp_query_default("pressure.dependency", value.pressure.arrhenius.dependency, 1); // Whether to use pressure to determined the reference Zeta 
+            pp_query_default("pressure.h1", value.pressure.arrhenius.h1, 1.81); // Surgate heat flux model paramater - Homogenized
+            pp_query_default("pressure.h2", value.pressure.arrhenius.h2, 1.34); // Surgate heat flux model paramater - Homogenized
 
         }
         else
         {
-            pp_query("pressure.r_ap", value.pressure.power.r_ap);     // AP power pressure law parameter (r*P^n)
-            pp_query("pressure.r_htpb", value.pressure.power.r_htpb); // HTPB power pressure law parameter (r*P^n)
-            pp_query("pressure.r_comb", value.pressure.power.r_comb); // AP/HTPB power pressure law parameter (r*P^n)
-            pp_query("pressure.n_ap", value.pressure.power.n_ap);     // AP power pressure law parameter (r*P^n)
-            pp_query("pressure.n_htpb", value.pressure.power.n_htpb); // HTPB power pressure law parameter (r*P^n)
-            pp_query("pressure.n_comb", value.pressure.power.n_comb); // AP/HTPB power pressure law parameter (r*P^n)
+            pp_query_default("pressure.r_ap", value.pressure.power.r_ap, 0.0);     // AP power pressure law parameter (r*P^n)
+            pp_query_default("pressure.r_htpb", value.pressure.power.r_htpb, 0.0); // HTPB power pressure law parameter (r*P^n)
+            pp_query_default("pressure.r_comb", value.pressure.power.r_comb, 0.0); // AP/HTPB power pressure law parameter (r*P^n)
+            pp_query_default("pressure.n_ap", value.pressure.power.n_ap, 0.0);     // AP power pressure law parameter (r*P^n)
+            pp_query_default("pressure.n_htpb", value.pressure.power.n_htpb, 0.0); // HTPB power pressure law parameter (r*P^n)
+            pp_query_default("pressure.n_comb", value.pressure.power.n_comb, 0.0); // AP/HTPB power pressure law parameter (r*P^n)
```

Comment:
@mongole584 I do not think these should be given as default values since they are garbage. Please double check.